modules/ballerina-mime/findbugs-exclude.xml

```diff
@@ -0,0 +1,20 @@
+<!--
```

Comment:
Do we need this ?

Reply:
You mean this file 'findbugs-exclude.xml'? All the modules do have this file even if there's no entry in it. Didn't want the mime package to be the exception.

Reply:
Even if we aren't filtering any of the packages/classes in the module, need to include this file. Otherwise when building, the findbugs plugin will complain about not being able to find the `findbugs-exclude.xml` file and will result in a build failure